app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,5 @@
+class PostsController < ApplicationController
+  def index; end
+
+  def show; end
```

Comment:
**[OPTIONAL]**

- [ ] It would be nice if you make a view file for this method because this method is giving an error of no HTML template 😨 Kindly add a HTML file for this method to make your project work as expected 👍